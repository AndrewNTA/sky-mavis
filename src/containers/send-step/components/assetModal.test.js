import React from 'react';
import renderer from 'react-test-renderer';
import AssetModal from './assetModal';

test('SendStep should render success modal', () => {
  const component = renderer.create(
    <AssetModal
      options={[{
        id: '12345',
        primaryValue: 10,
        primaryCurrency: 'USD',
        secondaryCurrency: 'VND',
        secondaryValue: 1000,
        icon: 'icon_url'
      }]}
      selectedId={'12345'}
      onSelect={() => {}}
      onClose={() => {}}
    />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});