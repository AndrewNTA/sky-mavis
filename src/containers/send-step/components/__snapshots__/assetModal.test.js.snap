// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendStep should render success modal 1`] = `
<div
  className="r-asset-modal"
>
  <div
    className="ram-overlay"
  />
  <div
    className="ram-main"
  >
    <div
      className="ram-header"
    >
      <span
        className="ram-header-title"
      >
        Assets
      </span>
      <img
        alt="close"
        className="ram-close-btn"
        onClick={[Function]}
        src="close-icon.png"
      />
    </div>
    <div
      className="ram-list"
    >
      <div
        className="ram-item selected"
        onClick={[Function]}
      >
        <img
          alt="USD"
          className="ram-currency-icon"
          src="icon_url"
        />
        <div>
          <div
            className="ram-group"
          >
            <span
              className="ram-primary-value"
            >
              10
            </span>
            <span
              className="ram-primary-currency"
            >
              USD
            </span>
          </div>
          <div
            className="ram-group"
          >
            <span
              className="ram-secondary-value"
            >
              1,000
            </span>
            <span
              className="ram-secondary-currency"
            >
              VND
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
